www/css/dark.css

```diff
@@ -14,25 +15,21 @@ body {
   background-color: var(--bg-color);
 }
 
```

Comment:
Shadows should be made darker

![image](https://user-images.githubusercontent.com/32238504/62171835-07fc6500-b339-11e9-8b81-f5b1fea8408a.png)

Reply:
Shadows? I can't find any shadows.
![light](https://user-images.githubusercontent.com/33164598/62176530-204f8e00-b2ff-11e9-87cd-cfdfbb276f27.png)
![dark](https://user-images.githubusercontent.com/33164598/62176542-2f364080-b2ff-11e9-984f-ab840b9b908c.png)

Reply:
Shadows are blocks that don't actually exist. An example is the string block in the send message block.

Reply:
Shadow selector is `.blocklyPath[fill='#bddbd1']`